src/D2L.CodeStyle.Analyzers/Threading/UseConfigureAwaitFalseAnalyzer.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace D2L.CodeStyle.Analyzers.Threading {
+
+	[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(UseConfigureAwaitFalseAnalyzerFixer)), Shared]
+	public class UseConfigureAwaitFalseAnalyzerFixer : CodeFixProvider {

```

Comment:
Could you place these in different files? And name this one `UseConfigureAwaitFalseFixer`.